.github/workflows/inclusive-language-check.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: Check inclusive language
+
+on:
+  workflow_call:
+
+jobs:
+  check-inclusive-language:
+    name: Check inclusive language
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: '0'
```

Comment:
I think we will need to use the repo name as well (like [this](https://github.com/vmware-tanzu/carvel-release-scripts/blob/96855a31d36ea25e9484764f18844c4f7023236d/.github/workflows/trivy-scan.yml#L43))

Reply:
I originally did call out the repo but after reading [the actions/checkout docs](https://github.com/actions/checkout) _I think_ it's unnecessary because the default value is appropriate:
```
# Repository name with owner. For example, actions/checkout
# Default: ${{ github.repository }}
repository: ''
```